seed/accounts/accounts.py

```
@@ -0,0 +1,23 @@
+
+from seed.accounts.api_tokens import seed_api_token
+from seed.accounts.buyer import seed_buyer
+from seed.accounts.licensee import seed_licensee
+from seed.accounts.module import seed_module
+from seed.accounts.seller import seed_seller
+from seed.accounts.user_group import seed_user_group
+
+logger = logging.getLogger(__name__)
+
+
+async def seed_accounts() -> None:  # noqa: WPS217
```

Ruff (0.14.7)

Unused `noqa` directive (unknown: `WPS217`)

Remove unused `noqa` directive

(RUF100)